script.js

```diff
@@ -0,0 +1,61 @@
+// DOM Element from HTML
+const bookForm = document.getElementById('book-form');
+const bookContainer = document.querySelector('.books');
+
+// Get an input from HTML by it's parent
+const inName = bookForm.book;
+const inAuthor = bookForm.author;
+
+// Create an array to push books on that
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+const addBooks = (book, author) => {
+  books.push({
+    book,
+    author,
+  });
+
```

Comment:
- [ ] (Optional) It would be nice to use the JavaScript method `concat` instead of `push`. This way, you would not mutate the original array when changing states as it is considered a best practice to not mutate the original array. 👍